pipeline/splicing/calcVarPriors.py

```diff
@@ -3444,65 +3428,112 @@ def getVarDict(variant, boundaries):
 
     return varDict
 
-def main():
-    parser = argparse.ArgumentParser()
-    parser.add_argument('-i', "--inputFile", default="built.tsv", help="File with variant information")
-    parser.add_argument('-o', "--outputFile", help="File where results will be output")
-    parser.add_argument('-v', "--variantFile", help="File containing protein priors for variants")
-    parser.add_argument('-b', "--boundaries", default="enigma",
-                        help="Specifies which boundaries ('enigma' or 'priors') to use for clinically important domains")
-    parser.add_argument('-g', "--genomeFile", help="Fasta file containing hg38 reference genome")
-    parser.add_argument('-t', "--transcriptFile", help="RefSeq annotation hg38-based genepred file")
-    args = parser.parse_args()    
-
-    inputData = csv.DictReader(open(args.inputFile, "r"), delimiter="\t")
+
+# Globals accessed in multi-processing function calc_one
+# REMIND: Use functools.partial to pass direction into the map function
+genome38 = None  # This is actually never used...
```

Comment:
can we remove it altogether?

Reply:
Went ahead and removed this global and the initial calling. Below that genome is passed through a ton of functions (all None and working) and I'd prefer to leave it for the bit refactor after this.